ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2118,3 +2099,39 @@ def test033_Duplicate_main_order_and_cahange_materiel_type(self):
         self.assertEqual(suborder_data['Material Type'], material_type)
         self.assertEqual(suborder_data['Article Name'], article)
         self.assertEqual(suborder_data['Test Units'], test_unit)
+
+    def test035_archived_test_unit_shoudnt_display_in_the_order_drop_down_list(self):
+        """
+        Orders: Archived Test unit: Archive Approach: Archived test units shouldn't appear in orders in the drop down list
+        LIMS-3710
+        :return:
+        """
+        self.test_unit_page.get_test_units_page()
+        self.base_selenium.LOGGER.info('select random test unit record')
+        testunits, payload = self.test_unit_api.get_all_test_units()
+        random_row = random.choice(testunits['testUnits'])
+
+        testunit_name = random_row['name']
+        self.order_page.apply_filter_scenario(filter_element='test_units:testunit_name_filter', filter_text=testunit_name,
+                                              field_type='text')
+        row = self.test_unit_page.get_last_test_unit_row()
+        self.order_page.click_check_box(source=row)
+        self.test_unit_page.archive_selected_test_units()
```

Comment:
can't we replace all above steps by define new function in test_unit_api that list all archived test units?

Reply:
done